GRAPHICS/SeethroughSetColorNear.md

```diff
@@ -1,12 +1,12 @@
 ---
 ns: GRAPHICS
-aliases: ["0x1086127B3A63505E"]
+aliases: ["0x1086127B3A63505E", "_SEETHROUGH_SET_COLOR_NEAR"]
```

Comment:
Alias here shouldn't be necessary due to it being the same name without a `_` prefix.